graphql/e2e/common/subscription.go

```diff
@@ -85,7 +86,6 @@ func NewGraphQLSubscription(url string, req *schema.Request, subscriptionPayload
 
 	msg := operationMessage{}
 	if err = conn.ReadJSON(&msg); err != nil {
-		conn.Close()
```

Comment:
Are we sure, we are not leaking the connection here? It is not obvious from the code.

Reply:
We probably are but this is a test and we are not doing any worse than what we were doing before this PR.